packages/material-ui-lab/src/SpeedDialAction/SpeedDialAction.js

```diff
@@ -103,7 +103,6 @@ class SpeedDialAction extends React.Component {
         placement={tooltipPlacement}
         onClose={this.handleTooltipClose}
         onOpen={this.handleTooltipOpen}
-        className={classes.root}
```

Comment:
This change is a fix regression for a pull request I merged too quickly without waiting for the visual regression tests.
IMHO we should rename the button classes root. But I think that it's for another pull request.

Reply:
> but for SpeedDialAction this would only be <Tooltip className="c1" />.

It should be `c1 c2` with the current implementation. The className isn't applied on the Tooltip.